backend/tests/api/test_auth.py

```
@@ -55,8 +55,8 @@ async def test_create_user__valid_data__user_created_and_logged_in(client, usern
 
     assert response.status_code == status.HTTP_201_CREATED
     body = response.json()
-    assert body.pop('id')
-    assert body == {'email': EMAIL, 'is_active': True, 'username': username, 'avatar_url': None}
+    assert 'access_token' in body and body['access_token']
```

Ruff (0.15.11)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)